tests/slate/test_hdg_poisson.py

```diff
@@ -93,7 +93,6 @@ def run_LDG_H_problem(r, degree, quads=False):
     return scalar_error, flux_error
 
 
-@pytest.mark.parallel
```

Comment:
Why is this removed?

Reply:
I don't recall, I'll drop it.

Reply:
Possibly leftover from debugging the test.